recipe/meta.yaml

```diff
@@ -36,11 +40,26 @@ requirements:
     - libpqxx =6.4
     - spdlog
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
-    - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
+outputs:
+  - name: "libhdbpp-timescale"
+    files:
+      - "lib/libhdb++timescale.so"
+      - "lib/libhdb++timescale.so.2"
+      - "lib/libhdb++timescale.so.{{ version }}"
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
+        - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
```

Comment:
This didn't run: `Nothing to test for: /home/conda/feedstock_root/build_artifacts/linux-64/libhdbpp-timescale-2.0.1-h924138e_2.conda`

I think you have to leave them before outputs for this package.